### PR TITLE
fix: get init script working for gradle >= v5.0

### PR DESCRIPTION
**Pull request by jochil**
### Motivation & Context
Fixes: https://github.com/CodeIntelligenceTesting/cifuzz/issues/550

### Description
Older gradle versions handled default values (for tests/reports) differently. In order to get it working this change makes sure that these values are set explicitly. 

Unfortunately some of the report properties we need (`outputLocation`, `required`, ...) are not supported by gradle 4.9... so we have to increase the minimum requirements to gradle 5.0 (which is just 4 month newer than 4.9)

### CODE REVIEW

**Comment by lukaswoellhaf** [comment]
Alsothere is a typo in the `init.gradle `file building a build directory named `jacaco`:

`def execFile = layout.buildDirectory.file("jacaco/cifuzz.exec").get().asFile` must be changed to:
`def execFile = layout.buildDirectory.file("jacoco/cifuzz.exec").get().asFile`

**Comment by jochil** [comment]
@lukaswoellhaf good catches, thanks a lot. The `.destination` properties are also working with recent gradle versions... that's nice. I updated the PR

You can find new installers here: https://github.com/CodeIntelligenceTesting/cifuzz/actions/runs/3854773627

File: docs/Installation-Guide.md [hunk 1]
```diff
@@ -70,3 +70,19 @@ rd /s %APPDATA%/cifuzz
...
+change on the `main` branch and can also be used on other branches too.
+
+To use one of these installers the following steps have to be executed:
+1. Download the `cifuzz_installer` artifact from the successful pipeline run 
+2. Unpack the artifact, for example with `unzip cifuzz_installer.zip` 
+3. Chose the installer for your operating system and run it
+```
+chmod u+x cifuzz_installer_linux
+./cifuzz_installer_linux
+```
```
Comment (MarkusZoppelt) [inline docs/Installation-Guide.md:88]: Why is this section part of this PR?

It's nice, it just might be better placed in another PR if it references another (internal?) issue.

Comment (jochil) [reply docs/Installation-Guide.md:88]: Just because I noticed this was missing when working on the task and preparing a pre-release for a user.
I can move it to another PR if you prefer

Comment (MarkusZoppelt) [reply docs/Installation-Guide.md:88]: I think it is fine to keep it here

**Review by MarkusZoppelt — APPROVED** [review]
